src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    getCategoryStatus: (state) => {
+      state.categories = 'under construction';
+    },
+  },
```

Comment:
- [ ] Please fix the categories `checkStatus` action as below.

```ruby
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },

Reply:
- Well resolved